### PR TITLE
Fix build failure for resolve dependency ext in LS

### PR DESCRIPTION
**Pull request by NipunaMarcus**
## Purpose
 - Fix build failure for resolve dependency ext in LS

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by sanjana — APPROVED** [review]
LGTM

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/BallerinaDocumentService.java [hunk 1]
```diff
@@ -412,7 +412,7 @@ public void init(LanguageServer langServer,
             }
 
             try {
-                Optional<Project> project = this.workspaceManager.project(filePath.get());
+                Optional<Project> project = this.workspaceManagerProxy.get(fileUri).project(filePath.get());
```
Comment (IMS94) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/BallerinaDocumentService.java:415]: @nadeeshaan Should we use `workspaceManagerProxy.get()` or `get(fileUri)` here? Other methods are using the `get()` method.

Comment (NipunaMarcus) [reply language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/BallerinaDocumentService.java:415]: true... we don't need the fileURI as this won't be access during the cloned project scenarios ... will fix this.

Comment (NipunaMarcus) [reply language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/BallerinaDocumentService.java]: fixed

**Review by NipunaRanasinghe — APPROVED** [review]

**Review by sanjana — APPROVED** [review]

**Review by IMS94 — APPROVED** [review]

**Pull request merged**